package com.michael.learnjava.cho4_exceptions;

public class TryCatchFinally {
    public static void main(String[] args) {
        //example3();
        int x = 4;
        // assert(x > 3);
        System.out.println();
    }

    private static void example3() {
        int x = 4;
        assert (x > 3);
    }
}
